### DISCUSSION TITLE
Pacstall v1.4 Turkuaz

### DISCUSSION THREAD

**Discussion by Elsie19** [pacstall/pacstall#79] [Releases]
#### Features
---
##### Additions
* You can run `pacstall -T <pkg>` to view a tree of the package files
* `pacstall -Qi` will output whether a package was installed as a dependency of another or if it was expicitly installed
* Update branches/repo are now persistent meaning that if you update to a different branch/repo, you will continue updating to that same branch when you run `pacstall -U`

##### Changes
* Pacstall now runs without `sudo` (it still uses root privileges internally)

#### Bugs
---
##### Bug Fixes
* Many, didn't count :p

##### Known bugs
* Pacstall isn't printing help if no flags are given [#27](https://github.com/pacstall/pacstall/issues/27)
* Multiple package inputs does not work

#### Additions
---
* Everything from Users
* Pacdeps can be used recursivly. An example would be that package A depends on B which depends on C.
#### Changes
---
* The metadata logger will record variables with an underscore (_) in front to avoid confusion with pacscript variables

<hr /><em>This discussion was created from the release <a href='https://github.com/pacstall/pacstall/releases/tag/1.4'>Pacstall v1.4 Turkuaz</a>.</em>

**Comment by 0oAstro** [comment]
WDYM Without sudo? It builds packages using fakeroot or it just uses sudo privilege for anything that requires installing, compiling and removing.

**Reply by Elsie19** [reply]
Sudo priveleges aren't needed to run pacstall. Packages still need to use sudo when appropriate. It won't run with fakeroot yet (that's for the 1.5 release)

**Reply by 0oAstro** [reply]
Oh ok, I would help in the fakeroot part.

**Reply by Elsie19** [reply]
> Oh ok, I would help in the fakeroot part. 

That would be great! Can you join the discord
